funcoeszz

```diff
@@ -768,6 +769,7 @@ zzzz ()
 				# $instal_msg
 				export ZZOFF=""  # desligue funcoes indesejadas
 				export ZZPATH="$ZZPATH"  # script
+				export ZZDIR="$ZZDIR"
```

Comment:
Aqui podia ter um comentário explicativo, igual à linha anterior:

```bash
export ZZDIR="$ZZDIR"  # pasta zz/
```

Reply:
Boa idéia, e ampliar isso para todos os shells.
Vou rever isso e acho até melhor colocar o comentário numa linha anterior, e não in-line. O que acha?